index.html

```diff
@@ -0,0 +1,233 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <title>Quiz</title>
+  <link rel="stylesheet" href="quiz.css">
+</head>
+<body>
+  <article id="Widgets">
+    <div id="cards">
+
+      <div class="card" id="widget-A">
+        <div class="card-header-block">
+          <header class="card-header">
+            Widget A
+          </header>
+          <br>
```

Comment:
`<br>`s aren't necessary when using block level elements as block level elements will take up an entire width, and the next element will immediately go to the next 'line'.